utils/common/src/main/java/brooklyn/util/yaml/Yamls.java

```diff
@@ -350,7 +351,11 @@ public String getMatchedYamlTextOrWarn() {
                 if (e instanceof KnownClassVersionException) {
                     log.debug("Known class version exception; no yaml text being matched for "+this+": "+e);
                 } else {
-                    log.warn("Unable to match yaml text in "+this+": "+e, e);
+                    if (e instanceof UserFacingException) {
+                        log.warn("Unable to match yaml text in "+this+": "+e.getMessage());
```

Comment:
not sure it would actually -- possibly trace but an error here should be easy to reproduce